packages/office-ui-fabric-react/src/components/Checkbox/examples/Checkbox.Basic.Example.tsx

```diff
@@ -75,6 +79,12 @@ export class CheckboxBasicExample extends React.Component<{}, ICheckboxBasicExam
           boxSide='end'
           styles={ styles }
         />
+
+        <Checkbox
```

Comment:
Can we move this to a separate example down below? We're wanting to keep the "basic" examples as cut and dry, and to spec, as possible.